patientsearch/src/js/components/patientList/PatientList.js

```diff
@@ -66,6 +73,7 @@ export default function PatientListTable() {
     return (
       <MyPatientsCheckbox
         label={getAppSettingByKey("MY_PATIENTS_FILTER_LABEL")}
+        checked={hasFlagForCheckbox("following")}
```

Comment:
this is great, @achen2401 .  should we use `?flags=MyPatientsCheckbox` rather than my rather ambiguous suggestion of `?flags=following` ?

Reply:
thanks @pbugni.  In ISACC, participants are referred to as "recipients" instead of "patients", so I think I am sticking with your suggestion of `flags=following` as that is more generic and fitting.